charts/files/config.toml

```diff
@@ -1385,5 +1383,5 @@ default_section = 'status'
 usual_reporters = [
     '@kim:sosnowkadub.de',
     '@gwmngilfen:matrix.org',
-    '@joshsimmons:matrix.org',
+    '@robinriley:matrix.org',
```

Comment:
we seem to be using a mix of trailing commas

Reply:
I'm unsure if the bot supports trailing commas at the end of arrays, though from a look at the config files it looks like we already have those for e.g. the governing board members.

Did you try this PR with a local deployment @HarHarLinks?

Reply:
I don't think this PR would work any differently from the current deployment since that already has the commas. I haven't spent any effort in testing this specifically though the toml spec says it's ok: https://toml.io/en/v1.0.0#array